### PR TITLE
ADSDEV-1307: Make oTracking instance available on the window client side

### PR DESCRIPTION
**Pull request by sami-ft**
We are changing how ads are served on FT.com. 

Ad specific behaviour will now be served externally rather than being bundled with the consuming app through `advertising/ads-display` as it's currently done. 

Part of this gradual change is to replicate current ad features to be compatible with this new way of serving ads on ft.com

The o-tracking instance is used by ads to improve user targeting, and notably, the rootID of the page is sent with ad request for matching and Ad operations purposes. 

Rather than including o-tracking into the Ads package [as it's currently done](https://github.com/Financial-Times/advertising/blob/main/packages/display/src/client/utils.js#L2), we want to make use of what's already downloaded and available on the consuming app. So that the root id can be retrieved and sent with the ad requests by calling `window.oTracking.getRootID()`. 

Since n-tracking is installed to all ft.com pages, it makes sense to make this change here rather than exporting the `oTracking` instance on the window for every app that makes up ft.com, but let me know if there's any concerns with the approach

 @jkerr321 @kavanagh @rowanmanning I added you to the review as you might have counterpoints to the approach, I'm very much open to doing something different if you think it could be problematic.

### CODE REVIEW

**Review by rowanmanning — COMMENTED** [review]
I'm comfortable with this approach from a Reliability perspective 👍 not a code-owner though so will let Platforms review

**Review by apaleslimghost — COMMENTED** [review]
one question, happy to approve if we can answer it either way

File: src/client/index.js
Comment (apaleslimghost) [inline src/client/index.js:43]: **suggestion**: could we include the old and new instances to help with debugging?
```suggestion
		console.warn("An oTracking instance already exists on window, skipping", { currentInstance: window.oTracking, ourInstance: oTracking });
```

**Review by apaleslimghost — APPROVED** [review]
one non-blocking suggestion, thanks for this!

**Pull request merged**
